src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/DateTimeToStringTransformerTest.php

```diff
@@ -94,6 +94,21 @@ public function testTransformWithDifferentTimezones()
         $this->assertEquals($output, $transformer->transform($input));
     }
 
+    public function testTransformDateTimeImmutable()
+    {
+        if (PHP_VERSION_ID < 50500) {
+            $this->markTestSkipped('This test requires PHP >= 5.5');
```

Comment:
I'd make these skip messages (this one and the other) more useful, eg.: DateTimeImmutable has been introduced in PHP 5.5